### PR TITLE
Retrain model for site data

### PR DESCRIPTION
**Pull request by StefanGerlach**
I retrained the model for site data, a video of the latest rosbag can be seen here:

https://drive.google.com/file/d/1TChbwH_Cm1tQt_3umaMwebpTYWDlCjhQ/view?usp=sharing

### CODE REVIEW

**Comment by stela** [comment]
Looks much better than before on the video. Green was quickly detected, it displays "red" though already when it's yellow, however that's good enough for stopping properly :-) Looks like good enough latency.

**Comment by stela** [comment]
Changes so far look good to me! But you had an improved model pending? Better merge it with that one instead then.

**Comment by StefanGerlach** [comment]
Yes, there is another version pending with that brighter images in training-set.

File: ros/src/tl_detector/site_traffic_light_config.yaml [hunk 1]
```diff
@@ -9,10 +9,10 @@ camera_preprocessing:
 stop_line_positions:
     - [8.0, 16.2]
 use_tensorflow_detection: True
-use_tensorflow_nth_frame: 2
+use_tensorflow_nth_frame: 6
```
Comment (stela) [inline ros/src/tl_detector/site_traffic_light_config.yaml:12]: Hope the latency is low enough with this setting, guess if predictions are really accurate we could lower it.

**Pull request merged**